script/Arcueid/Day 3/03_00_ARC03_5B.txt

```diff
@@ -1100,22 +1102,17 @@ She wouldn't be human if she were still alive after this.
 [sha:77b2d681ef46ce37927fdcf3c8e0b97602ef69ae]{
 -- Page 240, Offset 4639.
 -- なんでもなにもない。
-// Kinda a weird construction?
-// Is this supposed to be an expounded なんでもない -> I did it like it was nothing (kinda echoed in 4641)?
-// Or something about having no reason to do it?
-// First two lines also munged a bit for cleaner EN flow
-Did I really do this?
+But there's no question as to why.
 }
 [sha:867675ebdc35564048cbc9ed20b7a7660000944a]{
 -- Page 240, Offset 4640.
 -- たった今、自分の手で。
-Did %{i}I%{/i} really do this?
+Because just a few moments ago...
 }
 [sha:fc73677ee4ea07ef66f9932fb9e64e63a67de6b0]{
 -- Page 240, Offset 4641.
 -- 遠野志貴の手で、あっさりと、見知らぬ女をバラバラにしてしまったんじゃないか。
-Did I really just cut a woman I didn't even know to pieces,
- with my own hands, like it was nothing?
+Wasn't I the one who mutilated this stranger beyond recognition, and nobody else?
```

Comment:
Prefer original for this whole block here I think. Unless there's a strong reason for being painstakingly literal, I think it harms the text more than it helps.

Reply:
He asks "why", and it didn't follow up at all, and I felt like that takes away from
1. the intent
2. the pacing of the scene
3. the emphatic quality

Reply:
I was about to comment that it still doesn't really follow up on 'why', but realized the issue is that I'd been reading it as 'why [did I do this]' rather than 'why [is she dead]'. I doubt I'd be the only one to make that mistake, but not sure what can reasonably be done to clean it up. I suppose you could reify the state in 4693.

> "Wh――y?"
> But there's no question why she's dead.
> It just happened. I just %{i}made%{/i} it happen.
> I was the one who tore this stranger to pieces without a second thought.

Kinda pleased with this take on たった今、自分の手で。The complete thing does lose the か on the final sentence, but I feel like keeping it forces the construction to be a bit awkward, or means you have to backpropagate it to the lines before - 
> Didn't it just happen? Didn't I just %{i}make%{/i} it happen?
> Who else but me could have been the one to tear this stranger to pieces?